src/ocean/io/stream/Format.d

```diff
@@ -81,6 +81,7 @@ class FormatOutput(T) : OutputFilter
         private Layout!(T)      convert;
         private bool            flushLines;
 
+        deprecated("Use the 'format' method instead, or 'flush'")
```

Comment:
Actually I didn't do it on purpose, since `print` is already deprecated, using `opCall` *should* trigger the deprecation as well... Doesn't it ?

Reply:
It does, but it breaks after "remove all deprecated symbols" procedure I do on rebase.

Reply:
Remove the alias as well then ? :D

Reply:
Yes, which is why I mark alias deprecated explicitly here to - so that it becomes obvious and acknowledged, and not something I secretly do in my branch :)